modules/samples/product/MqttRetainSample/README.txt

```diff
@@ -0,0 +1,9 @@
+This sample client demonstrate how MQTT retain feature works with WSO2 message broker.
```

Comment:
can we rephrase ? 
for example "this sample client demonstrate the usage of MQTT retained messages ...